controllers/admin/AdminTabsController.php

```diff
@@ -381,4 +381,9 @@ public function ajaxProcessUpdatePositions()
             }
         }
     }
+
+    public function access($action, $disable = false)
+    {
+        return false;
+    }
```

Comment:
```suggestion
```

Can be removed, if you don't have authorization rules, you can't access to this page, but if someone want to tweak is own database in order to be able to reach this page he can, whereas with your system, he had to edit Core files, which is not a good idea